OTNetworkTest.h

```diff
@@ -0,0 +1,49 @@
+//
+//  OTNetworkTest.h
+//  Hello-World
+//
+//  Created by Sridhar on 22/05/15.
+//  Copyright (c) 2015 TokBox, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <OpenTok/OpenTok.h>
+#import "RCTBridgeModule.h"
+
+enum OTNetworkTestResult {
+    OTNetworkTestResultVideoAndVoice,
+    OTNetworkTestResultVoiceOnly,
+    OTNetworkTestResultNotGood
+};
+
+@protocol OTNetworkTestDelegate;
+
+@interface OTNetworkTest : NSObject
+{
+    
+}
+- (void)runConnectivityTestWithApiKey:(NSString*)apiKey
+                            sessionId:(NSString*)sesssionId
+                                token:(NSString*)token
+                   executeQualityTest:(BOOL)needsQualityTest
+                  qualityTestDuration:(int)qualityTestDuration
+                              resolve:(RCTPromiseResolveBlock)resolve
+                               reject:(RCTPromiseRejectBlock)reject
+                             delegate:(id<OTNetworkTestDelegate>)delegate;
```

Comment:
@adamtysonsmith `resolve` and `reject` need to be last args for this to work I believe https://facebook.github.io/react-native/docs/native-modules-ios.html#promises